FreshRSS/FreshRSS
# Added ability to mark articles as read on focus.

Closes #5723 

Changes proposed in this pull request:

- Allow a user to enable mark as read when focusing article.

How to test the feature manually:

1. Enable "Mark an article as read... when focused" under Configuration > Reading > Mark article as read.
2. Using shortcut "focus next without opening" confirm article is marked as read without opening.
3. Using shortcut "focus previous without opening" confirm article is marked as read without opening.

Pull request checklist:


Notes:

- Other mark as read attributes are not documented. More than happy to write some if we believe the option isn't clear to the end user.
- The automatic scrolling is a little jarring when using this feature. Out of scope of this PR but may consider further PR's to adjust in future.
